specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2025-03-03/gallery.json

```diff
@@ -2966,6 +3363,248 @@
       ],
       "description": "The definition of a parameter that can be passed to a custom action of a Gallery Application Version."
     },
+    "GalleryScript": {
```

Comment:
The guidance for the past few years from the SDK team has been not to flatten the properties.. pl remove